Modules/Libs/QuestieHash.lua

```diff
@@ -35,7 +35,8 @@ function QuestieHash:InitQuestLogHashes()
                 Questie._sessionWarnings[questId] = true
             end
         elseif (not isHeader) then
-            questLogHashes[questId] = QuestieHash:GetQuestHash(questId)
+            local oldHash = questLogHashes[questId]
+            questLogHashes[questId] = QuestieHash:GetQuestHash(questId) or oldHash
```

Comment:
oldHash is **always** nil --> questLog looks like there is no quest at all if there is cache miss